src/style.scss

```diff
@@ -0,0 +1,144 @@
+
+$button-color: #00acdc;
+$bg-color: white;
+$smalltext-color: #616070;
+$maintext-color: #060b35;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #d3d3d3;
+  background-color: $bg-color;
+  text-align: center;
+
+  .card {
+    padding: 16px;
+
+    &__image {
+      width: 160px;
+      height: 134px;
+      margin-top: 16px;
```

Comment:
dont use this margine, add correct padding for card
![image](https://github.com/mate-academy/layout_product-cards/assets/96265844/c18a8f6f-96c9-42ec-aaa2-a4325ac826f7)
![image](https://github.com/mate-academy/layout_product-cards/assets/96265844/6de36f25-96e8-406f-93b3-4b78fb81a54b)